my-first-homework.xml

```diff
@@ -0,0 +1,48 @@
+<dreamTeams>
+    <myDreamTeam>
+   <lakersTeam>
```

Comment:
Indentations have shifted a bit, but XML is still valid, we'll learn how to format it by one shortcut in IDE later ;)